internal/pkg/template/templates/workloads/services/lb-web/cf.yml

```diff
@@ -36,6 +36,13 @@ Parameters:
     Type: String
   TargetPort:
     Type: Number
+{{- if .NLB }}
+  NLBAliases:
+    Type: String
+    Default: ""
+  NLBPort:
```

Comment:
Why do we only expose aliases and port? How do we decide which fields of `template.NetworkLoadBalancerListener` we use parameter as opposed to go template?

Reply:
The initial motivation was simply to support grabbing NLB URL for the `URI()` function in `describe` package. To determine the url of the service, we need the port number for sure, and one of the aliases and the nlb's dns name. This is basically how the decision was made.